iac-extensions/arm/src/test/resources/checks/StringLiteralDuplicatedCheck/StringLiteralDuplicatedCheck.bicep

```diff
@@ -54,3 +54,40 @@ resource storageAccount 'Microsoft.Storage/storageAccounts' = {
   }
 }
 
+// Compliant: module paths cannot be variables
+module moduleName_one '../dosomething.bicep' = {
```

Comment:
This is not exactly the example from the ticket (added `../`), because for that example no issue was being raised. The user might've simplified the example without checking if it still raises, because an exclusion for `dosomething.bicep` has been in this rule from the very beginning.